packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -454,7 +465,7 @@ class _RenderableTileLayer extends _RenderableLayer<TileLayer> {
     }
   }
 
-  void _cacheHexagonalTiles() {
+  void _cacheIsometricStaggeredTiles() {
```

Comment:
Is there anything we can do to refactor these giant methods into something easier to read? Maybe even just putting these methods in a different file?

Reply:
I thought breaking them into individual functions was enough. Renderable layer could have subclasses maybe? I can look in another pr.